src/common/bind-events.js

```diff
@@ -57,11 +55,20 @@ function iterateEvents(target, entity, bindings, actionName) {
 }
 
 function bindEvents(entity, bindings) {
+  if (!entity || !bindings) { return this; }
+
```

Comment:
It's just style (although slightly smaller) but how about:
```
if (entity && bindings) {
  iterateEvents(this, entity, bindings, 'listenTo');
}
return this;
```

Reply:
meh.. I typically promote guards vs conditional logic if possible and this is more pattern-like with the `bindRequests` function.  FWIW I think much of this will be short-lived as it can be heavily refactored for v4 since we're removing the multi-handlers.. events and requests can share more of the logic